packages/react-components/src/theming/FluentThemeProvider.tsx

```diff
@@ -60,15 +61,28 @@ const ThemeContext = createContext<Theme>(defaultTheme);
 export const FluentThemeProvider = (props: FluentThemeProviderProps): JSX.Element => {
   const { fluentTheme, rtl, children } = props;
 
+  /**
+   * Pass only the children if we previously wrapped.
+   * This is to prevent the provider from being applied
+   * multiple times and wiping out configuration like RTL
+   * FluentThemeProvider
+   */
+  const alreadyWrapped = useProvider();
```

Comment:
this is called outside of the `<ProviderContext` - does this ever return true? My understanding the useContext has to be called from a component that is a child of `<ProviderContext />`

Reply:
oh i see how this works, never mind!